### DISCUSSION TITLE
How to use the MAX31865 and MAX6675 chips in micropython？

### DISCUSSION THREAD

**Discussion by Ethanlaucan** [micropython/micropython#16104] [Libraries & Drivers]
I want to use MAX31865 on Micropyton to read the temperature of PT100/PT1000, and the MAX6675 chip to read the temperature of thermocouples, but I have not been able to find some use cases. Can any friends provide some cases?

**Comment by sosi-deadeye** [comment]
You could record the temperature of the heating at the supply and return, calculate the difference and the heat energy emitted.

Temperature measurement is pretty boring, as there are never rapid changes and you always have to wait half an eternity for something to happen.

To make the task more interesting, you could produce a heating system with a two-point controller.

**Comment by peterhinch** [comment]
See [awesome micopython](https://awesome-micropython.com/#temperature-analog).

**Comment by Ethanlaucan** [comment]
Very good, thank you~


---- Replied Message ----
| From | Peter ***@***.***> |
| Date | 11/1/2024 00:11 |
| To | ***@***.***> |
| Cc | ***@***.***>,
***@***.***> |
| Subject | Re: [micropython/micropython] How to use the MAX31865 and MAX6675 chips in micropython？ (Discussion #16104) |

See awesome micopython.
